sys/include/net/sock/dtls.h

```diff
@@ -702,6 +702,10 @@ int sock_dtls_session_init(sock_dtls_t *sock, const sock_udp_ep_t *ep,
  *
  * @param[in] sock      @ref sock_dtls_t, which the session is created on
  * @param[in] remote    Remote session to destroy
+ *
+ * @note For tinyDTLS this function destroys the session object right after notifying the remote
+ *       peer about the closing. This is an interim solution, preventing endlessly blocked session
+ *       slots, but allows as a consequence truncation attacks.
```

Comment:
Maybe link the tinyDTLS issue here as well, so we can track this in case this somehow gets resolved: https://github.com/eclipse/tinydtls/issues/95

Reply:
Good idea! Squashed it right away.
I also watch the upstream changes to look out for fixes. A solution on tinyDTLS side would be the best of course. Just have to be careful later when supporting further DTLS stacks that they behave similarly.